packages/Uploader/server/upload.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
This is just for testing, right? Would that be more obvious if it were in the `tests/` folder?

Reply:
I think for this specific component which depended on a server having the folder at that level is ok.